redhat/pimd.spec

```diff
@@ -24,6 +24,7 @@ RFC 2362 with a few noted exceptions (see the RELEASE.NOTES for details).
 
 
 %build
+./autogen.sh
```

Comment:
The `autogen.sh` script should only be run when building directly from GIT. In released versions the configure script and Makefile.in files exist in the tarball.  Could you add some sort of "if configure does not exist; then autogen"?